src/pages/Homepage/BookCatalogueSection/BookCatalogueSection.module.css

```diff
@@ -156,6 +160,7 @@
   justify-content: center;
```

Comment:
```suggestion
```

Removing this aligns cards to the left, it looks a bit tidier than central alignment on smaller screens